tests/integration.rs

```diff
@@ -0,0 +1,17 @@
+use std::env;
+use std::process::{Command, Stdio};
+
+#[test]
+fn main() {
+    let path_result = env::current_dir();
+    let path = match path_result {
+        Ok(path) => path.display().to_string(),
+        Err(_) => panic!("Path is not found"),
+    };
+    let status = Command::new(path + "/target/x86_64-unknown-linux-gnu/debug/youki")
```

Comment:
If you want to use the pre-built one, why not simply merge the paths with youki?

Reply:
https://github.com/containers/youki/blob/adb43102e022d05c4730fa95e431b30050dec7d8/integration_test.sh#L9

I was just following your direct specification here, but if you want to just merge the youki paths, I'll do that!

Reply:
I think you just need to connect it to the `youki` path. That code is a bit out of date.